server/store/apps.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package store
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/server/utils"
+)
+
+func (s *store) StoreApp(app *App) error {
```

Comment:
We need a way to get "all installed apps". Do we have that in mind?

Reply:
yes, I filed https://mattermost.atlassian.net/browse/MM-29585 to address this